### PR TITLE
update ngrokConfig

### PR DESCRIPTION
**Pull request by AdrMXR**

### CODE REVIEW

File: Defs/ActionManager/Server/server_runner.py [hunk 1]
```diff
@@ -120,7 +123,10 @@ def start_localhost(port):
 
 def start_ngrok(port):
     ngrok_conf.PyngrokConfig(config_path=".config/ngrok.yml")
```
Comment (AdrMXR) [inline Defs/ActionManager/Server/server_runner.py:125]: This line generates an execution conflict, but I decided not to move anything to it

**Comment by sTiKyt** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DarkSecDevelopers/HiddenEye/pullRequest?prid=6060464&bid=19850534)

**Comment by sTiKyt** [comment]
Useless, run_ngrok method was deprecated and replaced with according class

**Comment by sTiKyt** [comment]
start_ngrok*

**Comment by sTiKyt** [comment]
Please next time read code before you modify it

**Comment by AdrMXR** [comment]
I tried the code and it worked fine, I wouldn't have uploaded it if it didn't work

**Comment by sTiKyt** [comment]
@AdrMXR Go to our telegram

**Comment by AdrMXR** [comment]
My telegram is AdrMXR

**Comment by AdrMXR** [comment]
> @AdrMXR Go to our telegram

I sent you telegram

**Comment by darkmidus** [comment]
So does this fix ngrok?

**Pull request merged**